ratis-client/src/main/java/org/apache/ratis/client/api/DataStreamApi.java

```diff
@@ -36,5 +38,5 @@
  */
 public interface DataStreamApi {
   /** Create a stream to write data. */
-  DataStreamOutput stream();
+  DataStreamOutputRpc stream();
```

Comment:
DataStreamApi is user API.  We want to hide the RPC API from users.  So, please don't make this change.

We may cast it to DataStreamOutputRpc in the test.

Reply:
Got it. Changed this back.